packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -256,15 +258,44 @@ export function buildRootObject() {
         issuerTable.getIssuerRecordByIssuer(issuer).brand,
       getAmountMath,
       makeZCFMint,
+      addEmptySeat: () => {
```

Comment:
Does `addEmptySeat` sound like it's adding one that already exists?

Should it be `makeEmptySeatKit`?

Reply:
I don't much like the name here for a user-facing method, but it fits our model for a `makeFooKit`, so done.